neo/rawio/intanrawio.py

```diff
@@ -576,43 +579,50 @@ def read_rhs(filename, file_format: str):
         for chan_info in channels_by_type[sig_type]:
-            name = chan_info["native_channel_name"]
             chan_info["sampling_rate"] = sr
             chan_info["units"] = "V"
             chan_info["gain"] = 0.0003125
             chan_info["offset"] = -32768 * 0.0003125
             chan_info["dtype"] = "uint16"
             ordered_channels.append(chan_info)
             if file_format == "header-attached":
+                name = chan_info["native_channel_name"]
                 data_dtype += [(name, "uint16", BLOCK_SIZE)]
             else:
                 data_dtype[sig_type] = "uint16"
 
     # 5: Digital input channel.
     # 6: Digital output channel.
     for sig_type in [5, 6]:
-        if len(channels_by_type[sig_type]) > 0:
-            name = {5: "DIGITAL-IN", 6: "DIGITAL-OUT"}[sig_type]
-            chan_info = channels_by_type[sig_type][0]
-            # So currently until we have get_digitalsignal_chunk we need to do a tiny hack to
-            # make this memory map work correctly. So since our digital data is not organized
-            # by channel like analog/ADC are we have to overwrite the native name to create
-            # a single permanent name that we can find with channel id
-            chan_info["native_channel_name"] = name  # overwite to allow memmap to work
-            chan_info["sampling_rate"] = sr
-            chan_info["units"] = "TTL"  # arbitrary units TTL for logic
-            chan_info["gain"] = 1.0
-            chan_info["offset"] = 0.0
-            chan_info["dtype"] = "uint16"
-            ordered_channels.append(chan_info)
-            if file_format == "header-attached":
-                data_dtype += [(name, "uint16", BLOCK_SIZE)]
-            else:
+        if file_format in ["header-attached", "one-file-per-signal"]:
+            if len(channels_by_type[sig_type]) > 0:
+                name = {5: "DIGITAL-IN", 6: "DIGITAL-OUT"}[sig_type]
+                chan_info = channels_by_type[sig_type][0]
+                # So currently until we have get_digitalsignal_chunk we need to do a tiny hack to
+                # make this memory map work correctly. So since our digital data is not organized
+                # by channel like analog/ADC are we have to overwrite the native name to create
+                # a single permanent name that we can find with channel id
+                chan_info["native_channel_name"] = name
+                chan_info["sampling_rate"] = sr
+                chan_info["units"] = "TTL"  # arbitrary units TTL for logic
+                chan_info["gain"] = 1.0
+                chan_info["offset"] = 0.0
+                chan_info["dtype"] = "uint16"
+                ordered_channels.append(chan_info)
+                if file_format == "header-attached":
+                    data_dtype += [(name, "uint16", BLOCK_SIZE)]
+                else:
+                    data_dtype[sig_type] = "uint16"
+        elif file_format == "one-file-per-channel":
```

Comment:
Maybe we add a comment over the else here just to say that one-file-per-channel stores actual 1 and 0 s for the digital in and out?

Reply:
Added